pages/common/templates.tsx

```diff
@@ -264,7 +264,7 @@ const ChartFrame = forwardRef(
     ref: React.Ref<HTMLDivElement>,
   ) => {
     return (
-      <div ref={ref} className={styles["chart-frame"]} style={{ height, width, overflow: "hidden" }}>
+      <div ref={ref} className={styles["chart-frame"]} style={{ height, width }}>
```

Comment:
From the [visual regression tests](https://github.com/cloudscape-design/chart-components/actions/runs/16048447291), it looks like the charts in the "Axes and thresholds" page got slightly taller. There is also one vertical axis label that wraps differently in the "Migration: Axes and scales" page.

Are these expected?